tetris.py

```diff
@@ -248,7 +285,7 @@ def block_rotator(self):
         if any(
             x not in range(game_width)
             or y >= game_height
-            or (x, y) in self.get_landed_coords()
+            or (x, y) in self.coord_extractor()  # This is so hard to conceive by myself is this an actual comprehension of some sort?
```

Comment:
This is of the form `any(long_condition for (x, y) in self.get_current_block())`. So we go through coords of current block, in a list comprehension (which isn't actually a list comprehension, since there's no `[` and `]`), and check whether the condition is true for any coords.

```suggestion
            or (x, y) in self.coord_extractor()
```